src/sas/sascalc/fit/BumpsFitting.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, history, max_step, pars, dof):
         # Depending on the time remaining, either display the expected
         # time of completion, or the amount of time remaining.  Use precision
         # appropriate for the duration.
-        if remaining_time >= 1800:
+        if remaining_time >= 18000:
```

Comment:
Maybe use the following:
```
# Completion is time is displayed as a delta (e.g., 36s, 2m 36s or 3h 12m) if completion time is less
# than delta_cutoff hours, or as HH:MM if it is within 10 hours, otherwise it is displayed using a
# full timestamp YYYY-mm-dd HH:MM.
delta_cutoff = 2 # hours before switching from duration to clock time
if remaining_time >= delta_cutoff*3600:
    ...
```
You could further add the following clause to the else if you wanted to display deltas more than one day away:
```
if remaining_time >= 24*3600:
    time = '%dd %dh'%(remaining_time//(24*3600), (remaining_time%(24*3600))//3600)
...
```

Reply:
Right. I was just lazy and did the minimum to fix the obvious typo. But I will revise the whole if block and reorganize it to put the time limits in increasing order, which is not the case now.
BTW, it has nothing to do with this, but if I spotted that is because I was trying to figure out why the reported chi2 in the log explorer is about twice higher than the real chi2 shown in the fit panel window when using mpfit. I explored why, but from what I could get this is coming from bumps/mpfit.py and bumps/fitters.py. I noticed that the number of dof is overestimated because the parameter_residuals are added to the standard residuals, but this does not explain the difference, so I guess there must be a factor of 2 somewhere else. This is only the case for mpfit, not for the other minimizers, so I'm not sure that I should open a sasview issue. Any suggestion?

Reply:
I created ticket https://github.com/bumps/bumps/issues/111